modules/enet/doc_classes/NetworkedMultiplayerENet.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		A PacketPeer implementation that should be passed to [member SceneTree.network_peer] after being initialized as either a client or server. Events can then be handled by connecting to [SceneTree] signals.
+		[b]Note:[/b] ENet only uses UDP, not TCP. When forwarding the server port to make your server accessible on the public Internet, you only need to forward the server port in UDP. You can use the [UPNP] class to forward the server port automatically when starting the server.
```

Comment:
I would add a **try** before `to forward the server port automatically` given UPNP won't work in so many cases (e.g. router does not support it, support for it is disabled, or local NAT is in place e.g. iptables).